tests/spread/core24/platforms/snaps/build-for-all/snap/snapcraft.yaml

```diff
@@ -0,0 +1,14 @@
+name: build-for-all
```

Comment:
We can add a test using `all` with grammar to make sure it works.

Reply:
I added spread tests and unit tests for this